pkg/storage/mock/storer.go

```diff
@@ -59,8 +62,19 @@ func (m *MockStorer) Put(ctx context.Context, mode storage.ModePut, chs ...swarm
 			}
 		}
 		m.store[ch.Address().String()] = ch.Data()
+		yes, err := m.Has(ctx, ch.Address())
+		if err != nil {
+			exist = append(exist, false)
+			continue
```

Comment:
Return error here?

Reply:
If err != nil , then only that case should be affected... we should go through other cases anyway as they can be true.

Reply:
In that case, an error that may be relevant is hidden. Given that this is the mock, it does not have to be that strict, the only reason for it would be to have more information on test failure  and during its debug process. It can stay as it is now.